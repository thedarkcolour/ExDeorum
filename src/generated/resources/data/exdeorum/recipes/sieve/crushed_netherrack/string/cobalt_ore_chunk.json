{
  "neoforge:conditions": [
    {
      "type": "neoforge:not",
      "value": {
        "type": "neoforge:tag_empty",
        "tag": "forge:ores/cobalt"
      }
    }
  ],
  "type": "exdeorum:sieve",
  "ingredient": {
    "item": "exdeorum:crushed_netherrack"
  },
  "mesh": "exdeorum:string_mesh",
  "result": "exdeorum:cobalt_ore_chunk",
  "result_amount": {
    "type": "minecraft:binomial",
    "n": 1.0,
    "p": 0.04
  }
}