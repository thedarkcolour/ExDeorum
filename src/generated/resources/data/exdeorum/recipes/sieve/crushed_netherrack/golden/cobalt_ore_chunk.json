{
  "neoforge:conditions": [
    {
      "type": "neoforge:not",
      "value": {
        "type": "neoforge:tag_empty",
        "tag": "forge:ores/cobalt"
      }
    }
  ],
  "type": "exdeorum:sieve",
  "ingredient": {
    "item": "exdeorum:crushed_netherrack"
  },
  "mesh": "exdeorum:golden_mesh",
  "result": "exdeorum:cobalt_ore_chunk",
  "result_amount": {
    "type": "minecraft:binomial",
    "n": 1.0,
    "p": 0.07
  }
}